### PR TITLE
facade call to findChildService requires an existing parent service, whi...

### PR DESCRIPTION
**Pull request by smousa**
...le Add/Update uses the DeploymentID

https://jira.zenoss.com/browse/CC-763

### CODE REVIEW

File: facade/service.go [hunk 1]
```diff
@@ -324,7 +324,16 @@ func (f *Facade) GetServiceEndpoints(ctx datastore.Context, serviceId string) (m
 func (f *Facade) FindChildService(ctx datastore.Context, serviceId string, childName string) (*service.Service, error) {
 	glog.V(3).Infof("Facade.FindChildService")
 	store := f.serviceStore
-	return store.FindChildService(ctx, serviceId, childName)
+	parentService, err := store.Get(ctx, serviceId)
```
Comment (kindkid) [inline facade/service.go]: Is it the parentService or the topLevelService?

Comment (smousa) [inline facade/service.go]: parentService, otherwise it would be called the tenant

Comment (kindkid) [inline facade/service.go]: is parentService the parent of the service with ID == serviceId? Or is it the exact service with ID == serviceId?

**Comment by wwwtyro** [comment]
test this

**Pull request merged**